src/CSI.IBTA.UserService/Services/UsersService.cs

```diff
@@ -0,0 +1,301 @@
+﻿using CSI.IBTA.DataLayer.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using CSI.IBTA.Shared.DTOs.Errors;
+using CSI.IBTA.Shared.Entities;
+using CSI.IBTA.Shared.Utils;
+using CSI.IBTA.UserService.Interfaces;
+using System.Net;
+using System.Security.Claims;
+
+namespace CSI.IBTA.UserService.Services
+{
+    internal class UsersService : IUsersService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public UsersService( 
+            IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task<GenericResponse<UserDto[]>> GetAllUsers(HttpContext httpContext)
+        {
+            await _unitOfWork.Accounts.All();
```

Comment:
EF should solve the mappings for you if you had map created, relations should not be solved manually.
Check this article: https://learn.microsoft.com/en-us/ef/core/modeling/relationships